index.html

```diff
@@ -618,6 +618,85 @@ <h4>
         </section>
       </section>
     </section>
+    <section>
+      <h2>
+        Automation
+      </h2>
+      <p>
+        The Device Posture API pose a challenge to test authors, as fully
+        exercising interface requires physical hardware devices. To address this
+        challenge this document defines a [[WEBDRIVER2]] [=extension commands=]
+        that allows users to control the reported device posture and simulate a
+        real device.
+      </p>
+      <h3>
+        Extension Commands
+      </h3>
+      <h4>
+        Set device posture
+      </h4>
+      <table class="simple">
+        <tr>
+          <th>
+            HTTP Method
+          </th>
+          <th>
+            [=extension command URI Template|URI Template=]
+          </th>
+        </tr>
+        <tr>
+          <td>
+            POST
+          </td>
+          <td>
+            deviceposture
```

Comment:
Please take a look at the WebDriver spec and get more familiar with the model. "deviceposture" is not a valid URI template: it doesn't start with "/" and it's not a subpath of "/session/{session id}" like pretty much every command. See the other specs we've been working on for the kind of URI template being used.

If you just say "/deviceposture", your ChromeDriver implementation wouldn't even be able to retrieve a corresponding session.

Reply:
Yep in code it was /session/{session id}/deviceposture, I write it that way also in this spec

Reply:
Missing text added in https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/d41ab3585ed56ea9826b7df8598cd60c8a5d1067